### PR TITLE
Add some randomness to node data

### PR DESCRIPTION
**Pull request by stevendanna**
If you PUT a node body with the same content, it doesn't get written to
the database, leading to a poor load test.

### CODE REVIEW

File: chef-client-run.go [hunk 1]
```diff
@@ -78,6 +78,13 @@ func chefClientRun(nodeClient chef.Client, nodeName string, runList []string, ge
 
 	time.Sleep(time.Duration(sleepDuration) * time.Second)
 
+	// Ensure that what we post at the end of the run is different frm previous nodes
```
Comment (jeremiahsnapp) [inline chef-client-run.go]: Can you please fix the typo and also make this say "from previous runs"? That makes more sense to me anyway.

Comment (stevendanna) [inline chef-client-run.go]: Yup

**Comment by stevendanna** [comment]
Typo fixed.

**Comment by jeremiahsnapp** [comment]
Thanks @stevendanna for adding this. I was planning to add something similar but I was actually going to just update the `ohai_time` attribute with the current time. Would you have any problem doing this instead of the random value `cache-buster`?

**Comment by stevendanna** [comment]
Yup, ohai_time makes more sense.  Will make that update in a bit.

**Comment by stevendanna** [comment]
Okie dokie, updated

**Comment by jeremiahsnapp** [comment]
Thanks a lot @stevendanna this is now in 0.2.0 via 3b146de7d2fd544fee8d2b43df319f1b42f763d2
Sorry it took so long

**Pull request closed without merging**